utils/svg_to_tsx.ts

```diff
@@ -0,0 +1,69 @@
+import fs from 'fs';
```

Comment:
Using an svg library would be better for maintainability

Reply:
Will be refactored later.